overlord/devicestate/export_test.go

```diff
@@ -168,6 +168,14 @@ func MockSnapstateInstallWithDeviceContext(f func(ctx context.Context, st *state
 	}
 }
 
+func MockSnapstateInstallPathWithDeviceContext(f func(st *state.State, si *snap.SideInfo, path, name string, opts *snapstate.RevisionOptions, userID int, flags snapstate.Flags, deviceCtx snapstate.DeviceContext, fromChange string) (*state.TaskSet, error)) (restore func()) {
+	old := snapstateInstallPathWithDeviceContext
```

Comment:
use `testutil.Backup` instead?

Reply:
Good point, changed now. Note though that there are still some mockers left in the file that would need to be changed, but that would be better to address in a follow-up.